ci/trigger_github.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+WORKFLOW="$1"
+COMMIT="$(git rev-parse HEAD)"
+ENVOY_REPO="${ENVOY_REPO:-envoyproxy/envoy}"
+
+echo "Trigger workflow (${WORKFLOW})"
+echo "  Branch: ${ENVOY_BRANCH}"
+echo "  Commit: ${COMMIT}"
+
+curl -sL \
```

Comment:
atm this fails ~silently as the workflow is not set up

we can either land the workflow separately first, or let it through and add error handling once it has landed